[UXE-2351] chore: migrate compliance to Docs (#840)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/compliance/pci-dss.mdx

---
title: Compliance PCI
description: Como prestadora de serviços PCI-DSS 4.0 Nível 1, a Azion mantém seus dados seguros em todas as suas edge locations sem comprometer a performance do seu site.
meta_tags: Azion, edge computing, compliance, cartões de pagamento, segurança e privacidade de dados
namespace: documentation_compliance_pci

> permalink: /documentacao/produtos/compliance/pci-dss-certification/

Author: ```suggestion
permalink: /documentacao/compliance/pci-dss-certification/
```

---

## Como isso afeta os clientes da Azion?

Como prestadora de serviços **PCI-DSS 4.0 Nível 1**, a Azion mantém seus dados seguros em todas as suas edge locations sem comprometer a performance do seu site.

Dessa forma, você pode:

- Simplificar auditorias PCI.
- Manter o nível mais alto de compliance.
- Aumentar a confiança do usuário.
- Processar credenciais de pagamento e outros dados sensíveis.
- Manter seu processo de check-out rápido e com alta performance.

---

## FAQ

- **Qual compliance a Azion conquistou?**

A Azion foi certificada como uma **prestadora de serviços PCI DSS 4.0 Nível 1**.

- **Quem realiza a auditoria independente da Azion para sua compliance PCI?**

A Azion contratou um [Qualified Security Assessor (QSA - Avaliadora de Segurança Qualificada)](https://www.pcisecuritystandards.org/assessors_and_solutions/qualified_security_assessors).

- **Quais regiões e produtos são cobertos?**

> A compliance PCI da Azion abrange toda a Plataforma de Edge Computing da Azion, o que inclui o Edge Application e todos os seus módulos (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões nas que a rede está implantada. Atualmente, a Azion está presente em seis continentes.

Author: ```suggestion
A compliance PCI da Azion abrange toda a Plataforma de Edge Computing da Azion, o que inclui o Edge Application e todos os seus add-ons (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus add-ons (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões nas que a rede está implantada. Atualmente, a Azion está presente em seis continentes.
```

> A compliance PCI da Azion abrange toda a Plataforma de Edge Computing da Azion, o que inclui o Edge Application e todos os seus módulos (Application Acceleration, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões nas que a rede está implantada. Atualmente, a Azion está presente em seis continentes.

Author: ```suggestion
A compliance PCI da Azion abrange toda a Plataforma de Edge Computing da Azion, o que inclui o Edge Application e todos os seus módulos (Application Accelerator, Edge Cache, Edge Functions e Load Balancer) e o Edge Firewall e seus módulos (DDoS Protection Unmetered, Network Layer Protection, Web Application Firewall - WAF e Origin Shield), cobrindo todas as regiões nas que a rede está implantada. Atualmente, a Azion está presente em seis continentes.
```

- **Por quanto tempo o Atestado de Compliance (AOC - Attestation of Compliance) da Azion é valido?**

O AOC atual da Azion foi emitido em *5 de maio de 2023 e é válido por um ano*.

- **Como posso obter uma cópia do Atestado de Compliance (AOC) da Azion?**

> Os clientes com plano de serviço Enterprise ou Mission Critical devem entrar em contato com seu Account Executive para solicitar um AOC. Para tanto, é necessário ter um NDA.

Reviewer: ```suggestion
Os clientes com plano de serviço **Enterprise** ou **Mission Critical** devem entrar em contato com seu Account Executive para solicitar um AOC. Para tanto, é necessário ter um NDA.
```

> Os clientes com plano de serviço Enterprise ou de Missão Crítica devem entrar em contato com seu Executivo de Conta para solicitar um AOC. Para tanto, é necessário ter um NDA.

Reviewer 2: ```suggestion
Os clientes com plano de serviço Enterprise ou MissionCritical vem entrar em contato com seu Account Executive para solicitar um AOC. Para tanto, é necessário ter um NDA.
```

Sobre account executive, não sei se traduz 🤔
